components/org.wso2.carbon.identity.authenticator.risk.analytics.endpoint/src/gen/java/org/wso2/carbon/identity/authenticator/risk/analytics/endpoint/factories/CalculateApiServiceFactory.java

```diff
@@ -0,0 +1,14 @@
+package org.wso2.carbon.identity.authenticator.risk.analytics.endpoint.factories;
+
+import org.wso2.carbon.identity.authenticator.risk.analytics.endpoint.CalculateApiService;
+import org.wso2.carbon.identity.authenticator.risk.analytics.endpoint.impl.CalculateApiServiceImpl;
+
+public class CalculateApiServiceFactory {
+
+   private final static CalculateApiService service = new CalculateApiServiceImpl();
+
+   public static CalculateApiService getCalculateApi()
+   {
```

Comment:
The above two lines should be in one line, shouldn't it?

Reply:
Refactored all swagger generated classes.